src/json/commons/data_decoded_change_master_copy.json

```diff
@@ -0,0 +1,10 @@
+{
+  "method": "changeMasterCopy",
+  "parameters": [
+    {
+      "name": "_masterCopy",
```

Comment:
always with `_`? 🤔 interesting

Reply:
Here is the link if you want to investigate more: https://safe-transaction.staging.gnosisdev.com/api/v1/safes/0x1230B3d59858296A31053C1b8562Ecf89A2f888b/all-transactions/?limit=20&offset=20

Reply:
I also don't get it :/

Reply:
it is a common practise in smart contract development to differentiate between the local variable and the parameter ;) but it is theoretically not required in the abi ... but no problem :)

Reply:
so should I be defensive when checking names? As in, checking with `contains` rather than exact `equals` ?

Reply:
no, I think our backend always returns the same. but this can also be improved with #55 so that we automatically try with and without `_` and maybe allow case insensitive matching or similar